bin/oneoff/fix_level_group_data

```diff
@@ -14,6 +14,8 @@
 #     ON user_levels.user_id = user_scripts.user_id
 #       AND user_levels.script_id = user_scripts.script_id
 #   WHERE
+#     user_levels.level_source_id IS NOT NULL
```

Comment:
There seem to be a number of these, so it simply seems simplest to avoid them when generating the data rather than programatically. And, in these cases, no action is required.